### PR TITLE
gitAtHead support

### PR DESCRIPTION
**Pull request by JamesDowney**
- Bumped KoLmafia to 4.0.1
- Swapped a call to `myPathId()` to `myPath()` in accordance with `votingBoothInitiatives()` change
- Removed ensuring Van & Duffel's Baleet Shop is a forbidden store
- Changed `checkGitVersion()` to utilize `gitAtHead()`
- Swapped to calling `checkGitVersion()` every time garbo is run

All feedback welcome, thank you for your hard work maintaining this wonderful script.

### CODE REVIEW

**Comment by JamesDowney** [comment]
Also, built and tested locally without issues. That's probably worth mentioning.

File: src/index.ts [hunk 1]
```diff
@@ -325,13 +324,7 @@ export function canContinue(): boolean {
 
 export function main(argString = ""): void {
   sinceKolmafiaRevision(26634);
```
Comment (JamesDowney) [inline src/index.ts:326]:
```suggestion
  sinceKolmafiaRevision(26713);
```

File: src/lib.ts [hunk 2]
```diff
@@ -377,20 +379,27 @@ export function safeRestore(): void {
...
   if (process.env.GITHUB_REPOSITORY === "CustomBuild") {
     print("Skipping version check for custom build");
   } else {
     const gitBranches: { name: string; commit: { sha: string } }[] = JSON.parse(
       visitUrl(`https://api.github.com/repos/${process.env.GITHUB_REPOSITORY}/branches`)
     );
-    const mainBranch = gitBranches.find((branchInfo) => branchInfo.name === "main");
-    const mainSha = mainBranch && mainBranch.commit ? mainBranch.commit.sha : "CustomBuild";
-    if (process.env.GITHUB_SHA === mainSha) {
+    const releaseCommit = gitBranches.find((branchInfo) => branchInfo.name === "release")?.commit;
```
Comment (horrible-little-slime) [inline src/lib.ts:392]: Can we move this logic & the gitBranches logic into the `else` statement?

**Review by horrible-little-slime — CHANGES_REQUESTED** [review]
1 tiny thing

File: src/lib.ts (hunk 2 above)
Comment (JamesDowney) [reply src/lib.ts:392]: Good call, yes, yes we can. One sec

**Pull request merged**